src/makeCalculator.js

```diff
@@ -4,7 +4,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculated = {
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+    operate: function (operation, number) {
+      this.result = operation(this.result, number);
```

Comment:
[CODE STYLE] - When modifying the `result` property, use the arithmetic operators with the assignment operator `=` for better code style and readability.

BAD EXAMPLE:
this.result = operation(this.result, number);

GOOD EXAMPLE:
// Assuming operation is this.add
this.result += number;

Note: This change requires a modification in how the `add`, `subtract`, `multiply`, and `divide` methods are defined and used.